ivy/func_wrapper.py

```diff
@@ -825,7 +825,6 @@ def _infer_device(*args, device=None, **kwargs):
 
 def handle_out_argument(fn: Callable) -> Callable:
     handle_out_in_backend = hasattr(fn, "support_native_out")
-    is_compos_fn = "ivy.functional.ivy" in fn.__module__
```

Comment:
We'd still need the `is_compos_fn` attribute right? If it's the compositional implementation, we should just pass the arguments as they are, without the need of an explicit `inplace_update` 🙂

Reply:
So, the idea is that the compositional implementation will not have any decorators that are only needed for the primary implementation ( e.g `handle_out_argument`) because we'll be adding these from the `add_wrappers` tuple inside the `_wrap_function`. And since, `handle_out_argument` will never be called with respect to the compositional implementation, there's no need for the `is_compos_fn` attribute.

Reply:
Ah makes sense. Thanks for clarifying!